src/tests.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Is this included erroneously?? Seems to be not used.

Reply:
WHOOPS. didn't mean to add this good catch